### ISSUE TITLE
Fix broken config-migration for 3.0.0-beta -> CONF_FRESH_WATER_MODULE not defined

### ISSUE THREAD

**Issue by lein1013** [LavermanJJ/home-assistant-solarfocus#46]
`CONF_FRESH_WATER_MODULE `not imported in` __init__.py`

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 629, in async_migrate
    result = await component.async_migrate_entry(hass, self)
  File "/config/custom_components/solarfocus/__init__.py", line 145, in async_migrate_entry
    new_options[CONF_FRESH_WATER_MODULE] = 0
NameError: name 'CONF_FRESH_WATER_MODULE' is not defined
```

**Comment by lein1013** [comment]
Pull Request opened https://github.com/LavermanJJ/home-assistant-solarfocus/pull/47

**Comment by LavermanJJ** [comment]
fixed by #47
